jetty-io/src/main/java/org/eclipse/jetty/io/WriteFlusher.java

```diff
@@ -104,29 +103,30 @@ protected WriteFlusher(EndPoint endPoint)
 
     /**
      * Tries to update the current state to the given new state.
+     *
      * @param previous the expected current state
-     * @param next the desired new state
+     * @param next     the desired new state
      * @return the previous state or null if the state transition failed
      * @throws WritePendingException if currentState is WRITING and new state is WRITING (api usage error)
      */
-    private boolean updateState(State previous,State next)
+    private boolean updateState(State previous, State next)
     {
-        if (!isTransitionAllowed(previous,next))
+        if (!isTransitionAllowed(previous, next))
             throw new IllegalStateException();
 
         boolean updated = _state.compareAndSet(previous, next);
         if (DEBUG)
-            LOG.debug("update {}:{}{}{}", this, previous, updated?"-->":"!->",next);
+            LOG.debug("update {}:{}{}{}", this, previous, updated ? "-->" : "!->", next);
         return updated;
     }
 
     private void fail(PendingState pending)
     {
         State current = _state.get();
-        if (current.getType()==StateType.FAILED)
+        if (current.getType() == StateType.FAILED)
```

Comment:
Please don't do reformat changes with substantial changes!  it makes it impossible to see what has changed!